openedx/core/djangoapps/user_api/views.py

```diff
@@ -95,6 +95,7 @@ def get(self, request):  # pylint: disable=unused-argument
 
         form_desc.add_field(
             "password",
+            required=False,
```

Comment:
Without this, the front-end will prevent you from submitting the form without a password. We shouldn't make them enter a password if we're just going to redirect them to web-auth anyway.